[main] ERROR io.micronaut.runtime.Micronaut - Error starting Micronaut server: Error processing bean [Definition: dispatch.Listener] method definition [void listen(InternalMessage message)]: Failed to retrieve a channel pool named [default] to register a listener
io.micronaut.context.exceptions.BeanContextException: Error processing bean [Definition: dispatch.Listener] method definition [void listen(InternalMessage message)]: Failed to retrieve a channel pool named [default] to register a listener
	at io.micronaut.context.AnnotationProcessorListener.onCreated(AnnotationProcessorListener.java:98)
	at io.micronaut.context.AnnotationProcessorListener.onCreated(AnnotationProcessorListener.java:44)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:2036)
	at io.micronaut.context.DefaultBeanContext.addCandidateToList(DefaultBeanContext.java:3141)
	at io.micronaut.context.DefaultBeanContext.getBeanRegistrations(DefaultBeanContext.java:3018)
	at io.micronaut.context.DefaultBeanContext.streamOfType(DefaultBeanContext.java:786)
	at io.micronaut.context.DefaultBeanContext.streamOfType(DefaultBeanContext.java:773)
	at io.micronaut.context.DefaultBeanContext.lambda$initializeContext$32(DefaultBeanContext.java:1615)
	at java.util.HashMap.forEach(HashMap.java:1336)
	at io.micronaut.context.DefaultBeanContext.initializeContext(DefaultBeanContext.java:1614)
	at io.micronaut.context.DefaultApplicationContext.initializeContext(DefaultApplicationContext.java:234)
	at io.micronaut.context.DefaultBeanContext.readAllBeanDefinitionClasses(DefaultBeanContext.java:2907)
	at io.micronaut.context.DefaultBeanContext.start(DefaultBeanContext.java:231)
	at io.micronaut.context.DefaultApplicationContext.start(DefaultApplicationContext.java:180)
	at io.micronaut.runtime.Micronaut.start(Micronaut.java:71)
	at dispatch.ApplicationKt.main(Application.kt:6)
Caused by: io.micronaut.messaging.exceptions.MessageListenerException: Failed to retrieve a channel pool named [default] to register a listener
	at io.micronaut.rabbitmq.intercept.RabbitMQConsumerAdvice.process(RabbitMQConsumerAdvice.java:124)
	at io.micronaut.rabbitmq.intercept.RabbitMQConsumerAdvice.process(RabbitMQConsumerAdvice.java:67)
	at io.micronaut.context.AnnotationProcessorListener.onCreated(AnnotationProcessorListener.java:95)
	... 15 common frames omitted
Caused by: io.micronaut.context.exceptions.BeanInstantiationException: Error creating connection to RabbitMQ
	at io.micronaut.rabbitmq.connect.RabbitConnectionFactory.connection(RabbitConnectionFactory.java:62)
	at io.micronaut.rabbitmq.connect.$RabbitConnectionFactory$Connection0Definition.build(Unknown Source)
	at io.micronaut.context.BeanDefinitionDelegate.build(BeanDefinitionDelegate.java:153)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:1979)
	at io.micronaut.context.DefaultBeanContext.createAndRegisterSingletonInternal(DefaultBeanContext.java:2770)
	at io.micronaut.context.DefaultBeanContext.createAndRegisterSingleton(DefaultBeanContext.java:2756)
	at io.micronaut.context.DefaultBeanContext.getBeanForDefinition(DefaultBeanContext.java:2427)
	at io.micronaut.context.DefaultBeanContext.getBeanInternal(DefaultBeanContext.java:2401)
	at io.micronaut.context.DefaultBeanContext.findBean(DefaultBeanContext.java:1285)
	at io.micronaut.context.BeanDefinitionDelegate.build(BeanDefinitionDelegate.java:142)
	at io.micronaut.context.DefaultBeanContext.doCreateBean(DefaultBeanContext.java:1979)
	at io.micronaut.context.DefaultBeanContext.createAndRegisterSingletonInternal(DefaultBeanContext.java:2770)
	at io.micronaut.context.DefaultBeanContext.createAndRegisterSingleton(DefaultBeanContext.java:2756)
	at io.micronaut.context.DefaultBeanContext.getBeanForDefinition(DefaultBeanContext.java:2427)
	at io.micronaut.context.DefaultBeanContext.getBeanInternal(DefaultBeanContext.java:2401)
	at io.micronaut.context.DefaultBeanContext.getBean(DefaultBeanContext.java:732)
	at io.micronaut.rabbitmq.intercept.RabbitMQConsumerAdvice.process(RabbitMQConsumerAdvice.java:122)
	... 17 common frames omitted
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at com.oracle.svm.jni.JNIJavaCallWrappers.jniInvoke_VA_LIST:Ljava_net_ConnectException_2_0002e_0003cinit_0003e_00028Ljava_lang_String_2_00029V(JNIJavaCallWrappers.java:0)
	at java.net.PlainSocketImpl.socketConnect(PlainSocketImpl.java)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:609)
	at com.rabbitmq.client.impl.SocketFrameHandlerFactory.create(SocketFrameHandlerFactory.java:60)
	at com.rabbitmq.client.impl.recovery.RecoveryAwareAMQConnectionFactory.newConnection(RecoveryAwareAMQConnectionFactory.java:62)
	at com.rabbitmq.client.impl.recovery.AutorecoveringConnection.init(AutorecoveringConnection.java:156)
	at com.rabbitmq.client.ConnectionFactory.newConnection(ConnectionFactory.java:1213)
	at com.rabbitmq.client.ConnectionFactory.newConnection(ConnectionFactory.java:1170)
	at com.rabbitmq.client.ConnectionFactory.newConnection(ConnectionFactory.java:1128)
	at com.rabbitmq.client.ConnectionFactory.newConnection(ConnectionFactory.java:1321)
	at io.micronaut.rabbitmq.connect.RabbitConnectionFactory.connection(RabbitConnectionFactory.java:59)
